lib/simple_form/components/errors.rb

```diff
@@ -13,6 +13,16 @@ def has_errors?
         object && object.respond_to?(:errors) && errors.present?
       end
 
+      def has_value?
+        return unless object && object.respond_to?(attribute_name)
```

Comment:
As we expect a boolean here, we can return `false`, What do you think?

Reply:
The whole method could be a a single operation:

    object && object.respond_to?(attribute_name) && object.send(attribute_name).present?

Reply:
@josevalim - Done ✅  https://github.com/plataformatec/simple_form/pull/1553/commits/12f38eeaa0ea392a7d3ba75796928437660a3577